packages/client/.vscode/settings.json

```diff
@@ -1,3 +1,16 @@
+/*
+👋 Hi! This file was autogenerated by tslint-to-eslint-config.
+https://github.com/typescript-eslint/tslint-to-eslint-config
+
+It represents the closest reasonable ESLint configuration to this
+project's original TSLint configuration.
+
+We recommend eventually switching this configuration to extend from
+the recommended rulesets in typescript-eslint. 
+https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/FAQs.md
+
+Happy linting! 💖
+*/
 {
     "typescript.tsdk": "./node_modules/typescript/lib"
```

Comment:
In all of the settings.json, can we follow the pattern we did with bolt? https://github.com/slackapi/bolt-js/blob/main/.vscode/settings.json

Essentially turning prettier to run on save.